deploy/updater/updater.py

```
@@ -4,329 +4,397 @@
+
+
+async def get_cache_data(mc, key_b, default_response=None):
+    if default_response is None:
+        default_response = {}
+
+    cache = await mc.get(key_b)
+
+    if cache:
+        cache = json.loads(cache.decode("utf-8"))
+    else:
+        cache = default_response
```

Ruff (0.8.2)

Use ternary operator `cache = json.loads(cache.decode("utf-8")) if cache else default_response` instead of `if`-`else`-block

Replace `if`-`else`-block with `cache = json.loads(cache.decode("utf-8")) if cache else default_response`

(SIM108)